app/controllers/my_tasks_controller.rb

```diff
@@ -16,13 +16,7 @@ def show_user_tasks
 
   # GET /my_tasks
   def index
-    @all_my_tasks = MyTask.all
-    @my_tasks = []
-    @all_my_tasks.each do | el |
-       @my_tasks << {my_task: el, tasks: el.task}
-    end
-
-    render json: @my_tasks
+    @my_tasks = MyTask.all
```

Comment:
Don't forget that this is development code, and will need to be refactored when auth is pushed and merged